src/BIMDataComponents/BIMDataMenu/BIMDataMenu.vue

```diff
@@ -128,8 +134,17 @@ export default {
       type: Boolean,
       default: false,
     },
+    /**
+     * This props is specific to the eponym platform feature.
+     * It's a temporary workaround that allow us
+     * to make items with no child hoverable.
+     */
+    userGroupImportFeature: {
+      type: Boolean,
+      default: false,
+    },
   },
-  emits: ["item-click"],
+  emits: ["on-hover"],
```

Comment:
`on-hover` ? It may sound like the name of the handler, not the event. The event may be `hover`, isn't it ?